src/test/java/net/masterthought/cucumber/ReportGenerator.java

```diff
@@ -17,20 +17,29 @@
 public abstract class ReportGenerator {
 
     public final static String JSON_DIRECTORY = "json/";
+    public final static String PROPERTY_DIRECTORY = "properties/";
```

Comment:
what about 

`public final static String CLASSIFICATIONS_DIRECTORY = "classifications/";` 
?

Reply:
Sounds ok to me

Reply:
actually should the name of this change actually? They are properties files first and foremost that get converted to classifications. Also I think folk would identify more so with properties first than classifications to be fair. Happy to use either or but just giving my two cents. I'll leave as is for now & we can update again if need be?

Reply:
Would be happy to see the name of the directory which tells me what it is used for and the content/files how should I read it. So change directory, please

Reply:
updated PR will fix this.